pkgs/applications/video/mpv/scripts/smart-copy-paste-2.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildLua,
+  fetchFromGitHub,
+  unstableGitUpdater,
+}:
+buildLua (finalAttrs: {
+  pname = "smart-copy-paste-2";
+  version = "unstable-2024-05-13";
```

```suggestion
  version = "0-unstable-2023-11-25";
```
This is the time of the last commit.

---

Done!